project1/pj1.html

```diff
@@ -14,7 +14,7 @@
             <div class="txt">
             <h1>PSYCHE CARE <br><span>Great PSD template for Psychology <br>
```

Comment:
Why is there a `<br>` is it structure or presentational? If later, you should use `display: block` to display span on new line